modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -2,7 +2,7 @@
 
 include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool-1.1.0.adoc[tag=intro]
 
-If the Runtime Manager agent is installed in your current Mule instance, the upgrade tool also updates the agent version as part of the upgrade process.
+If the Runtime Manager agent is installed in your current Mule instance, the upgrade tool also updates the agent version as part of the upgrade process. This process requires you to reinstall the Anypoint Monitoring agent. See xref:monitoring::am-installing.adoc#update-the-anypoint-monitoring-agent[Update the Anypoint Monitoring Agent].
```

Comment:
As Runtime Agent and Monitoring agent are different products, we might need to split the above statement into two different paragraphs. Stating that upgrading Anypoint Monitoring Agent is not supported, due to that, customer needs to reinstall it after a successful upgrade of Mule Runtime.